internal/metrics/metricsManager.go

```diff
@@ -124,10 +128,13 @@ type MetricsManager struct {
 	serviceLogicFilesystemUsagePercentage              *prometheus.GaugeVec
 }
 
-func NewMetricsManager(registry prometheus.Registerer) *MetricsManager {
+var _ API = &MetricsManager{}
```

Comment:
What is this line for?

Reply:
It a golang best practice to ensure MetricsManager always adhere to metric.API interface.